cmd/root.go

```diff
@@ -0,0 +1,87 @@
+package cmd
+
+import (
+	"github.com/appuio/image-cleanup/cfg"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+var (
+	// rootCmd represents the base command when called without any subcommands
+	rootCmd = &cobra.Command{
+		Use:              "image-cleanup",
+		Short:            "Cleans up images tags on remote registries",
+		PersistentPreRun: parseConfig,
+	}
+	config = cfg.NewDefaultConfig()
+)
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		log.WithError(err).Fatal("Command aborted")
```

Comment:
I'm much in favour of returning errors and let them bubble up instead of just pannicing. I'm open to discuss but I feel like this should be followed throughout the code basis.

Reply:
This is already the top level of execution. Or do you mean to bump it up to main even?

Reply:
Yeah I think main would make sense. But it was also more about all the other cases where `Fatal` is used.